nntrainer/models/neuralnet.cpp

```diff
@@ -199,6 +199,7 @@ int NeuralNetwork::compile() {
   NN_RETURN_STATUS();
 
   model_graph.topologicalSort();
+  setBatchSize(batch_size);
```

Comment:
any reason to set batch size here than at train directly? This changes a lot of semantics which we earlier set.

Reply:
changed.